src/SVGSkin.js

```diff
@@ -104,7 +104,7 @@ class SVGSkin extends Skin {
         return mip;
     }
 
-    updateSilhouette (scale = 1) {
+    updateSilhouette (scale = [100, 100]) {
```

Comment:
:flushed: 

(fortunately, this didn't break anything--all call sites specified a scale explicitly)